tracing-example/pom.xml

```diff
@@ -227,24 +219,6 @@
                         <version>${version.wildfly-maven-plugin}</version>
                         <configuration>
                             <jboss-home>${jboss.home}</jboss-home>
-                            <!-- First configure the feature pack we are overriding -->
-                            <feature-packs>
-                                <feature-pack>
-                                    <groupId>org.wildfly</groupId>
-                                    <artifactId>wildfly-ee-galleon-pack</artifactId>
-                                    <version>${version.org.wildfly}</version>
-                                    <excluded-packages>
-                                        <name>product.conf</name>
-                                        <name>docs.schema</name>
-                                    </excluded-packages>
-                                </feature-pack>
-                                <!-- Next include our own feature pack -->
-                                <feature-pack>
-                                    <groupId>org.jboss.resteasy</groupId>
-                                    <artifactId>galleon-feature-pack</artifactId>
-                                    <version>${version.org.jboss.resteasy}</version>
-                                </feature-pack>
-                            </feature-packs>
```

Comment:
If we remove this, we don't override the RESTEasy version in the container. Maybe that's not a big deal, but something to consider. TBH it's something we should think about for all the examples.

Reply:
Okay I see!